### ISSUE TITLE
Windows version svn

### ISSUE THREAD

**Issue by hoseyhosey** [jgranick/md-haxebinding#3]
The plugin folder requires Mono.Posix.dll V4 found in
http://ftp.novell.com/pub/mono/archive/2.10.2/windows-installer/5/mono-2.10.2-gtksharp-2.12.10-win32-5.exe
for the options to be available.

When typing a command prompt continually pops up for each key stroke.

Project creation is slow...but it appears to work correctly.

Cheers,
Good luck on this project

**Comment by jgranick** [comment]
When you say that the "Mono.Posix.dll" plugin is required, is that for the project options to work?

On Windows I did install (as instructed in the download page) GTK sharp as well as MonoDevelop. I seem to have been able to run it without doing anything else which is special, but I do see the command windows appearing.

I've gotten pretty frustrated trying to get them to disappear. For some reason it does not resolve the references in the library's source code when I'm on Windows, so I was having to make changes, build and copy from Linux over to Windows. Not a nice way to test things.

There surely must be a way to get the command prompt to disappear, but I'm not sure what more needs to be done right now...

**Comment by jgranick** [comment]
This issue was just fixed. There is more lag on Windows when calling "haxe --display", unfortunately. I am guessing it may be related to calling a separate process in general... perhaps there's more overhead?

On Linux the code completion is appearing instantly. I can feel hesitation in the editor when it is processed on Windows

**Comment by hoseyhosey** [comment]
Newest update removed command window appearing.

Both machine I tested on were windows 7, 64 bit.  Both required the mono.posix.dll.  I installed the GTK sharp (MonoDevelop wont install if that is not installed.

If you go to 
Tools->Options and select Haxe it throws an error
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.IO.FileNotFoundException: Could not load file or assembly 'Mono.Posix, Version=4.0.0.0, Culture=neutral, 

I found the mono.posix v4 in the provided url.  I had to install from the URL then find the dll.  At which point I was able to uninstall mono.

I am not getting a completion window as well.
